xyz-hub-service/src/main/resources/openapi-recipes.yaml

```diff
@@ -120,7 +120,7 @@ recipes:
     replace:
       - type: value
         path: servers.0.url
-        replace: /
+        replace: http://localhost:8080/
```

Comment:
When being deployed on a different server, will this work as well?

Reply:
yes, the new router included an URL-format validation, however it does not influence in the routing.